shared/chat/conversation/messages/text/index.js

```diff
@@ -4,22 +4,52 @@ import * as Types from '../../../../constants/types/chat2'
 import * as Kb from '../../../../common-adapters'
 import * as Styles from '../../../../styles'
 
+type ReplyProps = {
+  edited: boolean,
+  onClick: () => void,
+  text: string,
+  username: string,
+}
+
+const Reply = (props: ReplyProps) => {
```

Comment:
Could this say if the replied-to message exploded or was deleted?

Reply:
It leaves a placeholder for exploded, and is gone for deleted.

Reply:
Fixed